moses/moses/main/populate_atomspace.cc

```diff
@@ -0,0 +1,62 @@
+#include <moses/data/table/table.h>
+#include <moses/data/table/table_io.h>
+#include <opencog/atomspace/AtomSpace.h>
+#include <moses/utils/value_key.h>
+#include <opencog/atoms/base/Link.h>
+#include "populate_atomspace.h"
+#include <opencog/atoms/value/Value.h>
+
+namespace opencog { namespace moses {
+
+void populate(AtomSpace *as, ITable &itable) {
+
+	std::vector<multi_type_seq>::const_iterator it, end;
```

Comment:
It seems to me `it` and `end` are unused.

Reply:
You are right I forgot about that one

Reply:
Just fixed it